src/components/Header/index.tsx

```diff
@@ -46,6 +51,8 @@ export default function Header() {
   return (
     <>
       <header className={styles.header}>
+        {headerBarTitle && headerBarHref && <HeaderBar href={headerBarHref}>{headerBarTitle}</HeaderBar>}
+        {headerBarTitle && !headerBarHref && <HeaderBar>{headerBarTitle}</HeaderBar>}
```

Comment:
Since  `href` is optional can't we just check for the title?

```suggestion
        {headerBarTitle && <HeaderBar href={headerBarHref}>{headerBarTitle}</HeaderBar>}
```

Reply:
you are right, thanks! I will change it